webserver.cpp

```diff
@@ -596,7 +598,12 @@ int8_t http_parse_request(struct webserver_t *client, uint8_t **buf, uint16_t *l
       }
     }
     if(client->substep == 4) {
-      unsigned char *ptr = (unsigned char *)memchr(client->buffer, ':', client->ptr);
```

Comment:
Again another change to stop looking for ':' header seperators beyond the header double CRLF ending (which is set as posEnd). All changes further below in this function is also for this.